deploy.bat

```diff
@@ -1,14 +1,20 @@
 REM Scripts to build .net driver and deploy
 SET VERSION=%1
 SET API_KEY=%2
+SET SNKEY=%3
 
 SET ROOT_DIR=%~dp0 
 cd %ROOT_DIR%
 
-dotnet build Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n 
+echo -----BEGIN CERTIFICATE----- > "C:\jenkins\workspace\NugetPushDotNetDriverSignTest\coded.txt"

```

Comment:
Should we change the name of the directory and remove the `Test` or maybe even use relative path instead of the absolute? E.g. `..\coded.txt` instead of the whole path?

Reply:
@sfc-gh-pbulawa Thanks for checking , have done the changes to %WORKSPACE%